pinot-common/src/test/java/org/apache/pinot/util/TestLikeSyntaxConverter.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.util;
+
+import org.apache.pinot.common.utils.LikeToRegexFormatConverterUtil;
+import org.testng.annotations.Test;
+
+
+/**
+ * Tests for {@LikeToRegexFormatConverterUtil}
+ */
+public class TestLikeSyntaxConverter {
+
+  private static final String TRAILING_WILDCARD = "C+%";
```

Comment:
Can we add more test cases? Currently there is no test case with single character pattern, or wildcard in the middle